PSReadLine/Completion.cs

```diff
@@ -483,16 +483,24 @@ void EnsureMenuAndInputIsVisible(IConsole console, int tooltipLineCount)
                 }
             }
 
-            public void DrawMenu(Menu previousMenu)
+            public void DrawMenu(Menu previousMenu, bool menuSelect)
             {
                 IConsole console = Singleton._console;
 
                 // Move cursor to the start of the first line after our input.
                 this.Top = Singleton.ConvertOffsetToPoint(Singleton._buffer.Length).Y + 1;
-                EnsureMenuAndInputIsVisible(console, tooltipLineCount: 0);
+                if (menuSelect)
+                {
+                    EnsureMenuAndInputIsVisible(console, tooltipLineCount: 0, rows:this.Rows);
+
+                    console.CursorVisible = false;
+                    console.SaveCursor();
+                }
+                else
+                {
+                    EnsureMenuAndInputIsVisible(console, tooltipLineCount: 0, rows:0);
```

Comment:
Is this call necessary? I suspected it wasn't and tested briefly with it removed, I didn't see any problems.

Reply:
@jianyunt - after resolving this comment (either removing the call or confirming it is needed), I think this PR is ready to merge.

Reply:
@lzybkr Yes we need to keep this Line https://github.com/lzybkr/PSReadLine/blob/2f57fe5091e0e0cd343b86851dc31c9bdcb2afd5/PSReadLine/Completion.cs#L501  to avoid the ArgumentOutOfRangeException exception.

Reply:
Are counting on the `+ 1` to write out a single newline?

Based on the comment in `EnsureMenuAndInputIsVisible`, I could see that `+ 1` being removed at some point, and then the method would do nothing, right?

If my understanding is correct, it would be good to have some way to ensure this problem doesn't come back if someone removes that `+ 1` - a test, or maybe pass `rows:1`.